src/pint/models/stand_alone_psr_binaries/DDK_model.py

```diff
@@ -251,14 +257,14 @@ def delta_omega_proper_motion(self):
         """The correction on omega (Longitude of periastron)
         due to the pulsar proper motion
         (KOPEIKIN. 1996 Eq 9.)
-        d_omega = csc(i) * (PMRA_DDK * cos(KOM) + PMDEC_DDK * sin(KOM)) * (t-T0)
+        d_omega = csc(i) * (PMLONG_DDK * cos(KOM) + PMDEC_DDK * sin(KOM)) * (t-T0)
```

Comment:
Also here: shouldn't `PMDEC_DDK` be `PMLAT_DDK`?

Reply:
Same